examples/image-streams/image-streams-centos7.json

```diff
@@ -14,7 +14,11 @@
         "dockerImageRepository": "openshift/ruby-20-centos7",
```

Comment:
yeah i was debating whether to leave that or not...  if i remove it, nothing (ie tags we don't specify explicitly) will get automatically imported, i'm not sure that's what we want.  so i think it's appropriate to leave it.